src/esp/gfx/Simulator.cpp

```diff
@@ -53,6 +53,9 @@ void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
   }
 
   std::string houseFilename = io::changeExtension(sceneFilename, ".house");
+  if (!io::exists(houseFilename)) {
```

Comment:
Is this still needed? You can avoid this by adding a "house" entry to your configuration in cfg.scene.filepaths

Reply:
@msbaines by default, we are not filling a "house" entry in cfg.scene.filepaths. To support same behavior for `".house" (line 55) and ".scn" that make sense to leave it. Otherwise, wider refactoring needed and there will be no that seamless experience loading MP3D with semantic scenes.